src/EllCrv/Finite.jl

```diff
@@ -1148,17 +1148,17 @@ Return a list of generators of the group of rational points on $E$.
 julia> E = elliptic_curve(GF(101, 2), [1, 2]);
 
 julia> gens(E)
-2-element Vector{EllCrvPt{fqPolyRepFieldElem}}:
- Point  (93*o + 10 : 22*o + 69 : 1)  of Elliptic curve with equation
+2-element Vector{EllCrvPt{FqFieldElem}}:
+ Point  (16*o + 42 : 88*o + 97 : 1)  of Elliptic curve with equation
 y^2 = x^3 + x + 2
- Point  (89*o + 62 : 14*o + 26 : 1)  of Elliptic curve with equation
+ Point  (88*o + 23 : 94*o + 22 : 1)  of Elliptic curve with equation
```

Comment:
Are these changes here mathematically correct?

Reply:
The output is randomized, and the random generator will be different for the new finite field. I will probably double check with tomorrow.

Reply:
Yes, it is correct.